how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,171 @@
+```bash
+$ sudo install -D -o ${USER} -g ${USER} -m 0640 /usr/share/containers/libpod.conf ~/.config/containers/
+```
+
+By default the `tmp_dir` in `libpod.conf` is set to `/var/run/libpod`, however that is not accessible by a user, so change to the rootless runtime directory.
+```bash
+$ sed -i -e "s|^tmp_dir = .*$|tmp_dir = \"$XDG_RUNTIME_DIR/libpod/tmp\"|" ~/.config/containers/libpod.conf
+```
+
+#### 8. Add Kata Runtime to Podman configuration file (optional)
+You can tell Podman to create or run containers using the Kata runtime with `--runtime=value` flag.
+
+You can directly pass the fully qualified Kata runtime path with:
+```bash
+$ podman run --runtime=/usr/local/bin/kata-runtime ...
+```
+
+or add a `kata` entry in the `[runtimes]` section of the configuration file by appending the Kata Runtime binary path(s) to the `libpod.conf` file:
+```bash
+$ echo 'kata = ["/usr/local/bin/kata-runtime"]' >> ~/.config/containers/libpod.conf
+```
+and then use `kata` as the runtime name:
+```bash
+$ podman run --runtime=kata ...
+```
+
+> **NOTE:**
+> A less recommended approach could be to have the absolute `kata-runtime` path in the standard `$PATH` location instead of the configuration file, and a binary with that name will be looked up automatically:
+```bash
+kata-runtime = [
+]
+```
+
+#### 9. Set Kata runtime as Podman's default OCI runtime (optional)
+To avoid using the `--runtime=value` flag set Kata runtime as the default runtime:
+```bash
+$ sed -i -e 's/^runtime = "runc"/runtime = "kata"/' ~/.config/containers/libpod.conf
+```
+
+## Run Kata with rootless Podman
+```bash
+$ podman run --rm --runtime=kata alpine date
+```
+
+> **NOTE:**
+> To obtain debug logs you can:
+>  - Enable [debug](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug) in Kata (logs are added to journald).
+>  - Pass `--log-level=debug` to Podman (logs are printed to stderr).
+
+## Appendix: Possible Errors
+If you are building from source you may encounter the following errors.
+
+### Error caused by agent or runtime version mismatch
+```
+rpc error: code = Internal desc = Could not add route dest()/gw(10.0.2.2)/dev(tap0): network is unreachable: OCI runtime error
+```
+Solution:
+You may need to [rebuild the agent](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#add-a-custom-agent-to-the-image---optional); there was a change in both the [agent](https://github.com/kata-containers/agent/commit/a78e8cfda627cc350dc9d9ca9b969ebb642030c3) and [runtime](https://github.com/kata-containers/runtime/commit/cfedb06a19135e2ab4f18203a4f3147cdc3a4980) code. This would probably only occur if building latest from source, since the runtime version would have the change and the released agent does not.
+
+### Missing registry file
+```
+Error: unable to pull alpine: image name provided is a short name and no search registries are defined in the registries config file.
```

Comment:
Suggested rewrite: 
Error: unable to pull alpine: image name provided is a short name. The registries config file does not define the search registries.

Reply:
Hi @klynnrif - alas, we can't change this here as the doc is simply reporting an error (verbatim) coming from the runtime itself. The PR is now over at https://github.com/kata-containers/documentation/pull/565.